src/apis/status.ts

```diff
@@ -9,26 +55,96 @@ import chalk from 'chalk';
  * - Currently running local networks/nodes.
  * - Account information.
  * - Useful commands.
+ *
+ * @param env The name of the environment we are showing status for
+ * @param suppressLogging If truthy, then nothing will be printed to the console.
  */
-export function cli() {
-  console.log(chalk.cyan('\n[Boson]'));
-  console.log('Version 0.1.0');
-  console.log('Backend: Geth 1.0.0 - Web3 1.0.0b3');
+export async function cli(env: string, suppressLogging?: boolean): Promise<IProjectStatus> {
```

Comment:
i think cli should be sync. if this needs to be async maybe wrap it with something else?

also when should suppress logging be true? i'm thinking if we set different winston loggers and configure them globally that is probably cleaner. also helps with #23 .

Reply:
Suppress logging flag is actually used to suppress listr outputs, it is only true in tests. `cli` is async because in tests I want a data structure that holds the data `status` pulls out of the current environment... How would we wrap this and still provide that data for testing? I do agree that it being async is not the best.

Reply:
By `status` I just mean the `status.ts` module. I guess I could pass a flag to `cli()` that prints the output either in human readable form or as json, and for testing I'll just parse the json output.